### ISSUE TITLE
{} of method body ejected on the next line

### ISSUE THREAD

**Issue by sjrd** [olafurpg/scalafmt#305]
Original:
```scala
object RTCIceCandidateInit {
  @inline
  def apply(
      candidate: js.UndefOr[String] = js.undefined,
      sdpMid: js.UndefOr[String] = js.undefined,
      sdpMLineIndex: js.UndefOr[Double] = js.undefined): RTCIceCandidateInit = {
    val result = js.Dynamic.literal()
    candidate.foreach(result.candidate = _)
    sdpMid.foreach(result.sdpMid = _)
    sdpMLineIndex.foreach(result.sdpMLineIndex = _)
    result.asInstanceOf[RTCIceCandidateInit]
  }
}
```

Scalafmt:
```scala
object RTCIceCandidateInit {
  @inline
  def apply(candidate: js.UndefOr[String] = js.undefined,
      sdpMid: js.UndefOr[String] = js.undefined,
      sdpMLineIndex: js.UndefOr[Double] = js.undefined): RTCIceCandidateInit =
    {
      val result = js.Dynamic.literal()
      candidate.foreach(result.candidate = _)
      sdpMid.foreach(result.sdpMid = _)
      sdpMLineIndex.foreach(result.sdpMLineIndex = _)
      result.asInstanceOf[RTCIceCandidateInit]
    }
}
```

I would like the input to look like this:
```scala
object RTCIceCandidateInit {
  @inline
  def apply(candidate: js.UndefOr[String] = js.undefined,
      sdpMid: js.UndefOr[String] = js.undefined,
      sdpMLineIndex: js.UndefOr[Double] = js.undefined): RTCIceCandidateInit = {
    val result = js.Dynamic.literal()
    candidate.foreach(result.candidate = _)
    sdpMid.foreach(result.sdpMid = _)
    sdpMLineIndex.foreach(result.sdpMLineIndex = _)
    result.asInstanceOf[RTCIceCandidateInit]
  }
}
```

Using:
* 0.2.6-RC1
* --style Scala.js

The problem is that the `{}` of the method body is somehow ejected on the next line instead of staying attached to the `=`. I suspect the 80 column limit to be located precisely between the `=` and `{` ^^

**Comment by olafurpg** [comment]
Huh, that's very surprising. Should be a straightforward fix.

**Comment by olafurpg** [comment]
Fixed by f3544a9, referenced wrong issue from commit.
